projects/storefrontlib/src/layout/layout.module.ts

```diff
@@ -2,9 +2,10 @@ import { NgModule } from '@angular/core';
 import { Config } from '@spartacus/core';
 import { OutletRefModule } from '../cms-structure/outlet/outlet-ref/outlet-ref.module';
 import { LayoutConfig } from './config/layout-config';
+import { LaunchDialogModule } from './launch-dialog/index';
 
 @NgModule({
-  imports: [OutletRefModule],
+  imports: [OutletRefModule, LaunchDialogModule.forRoot()],
```

Comment:
I was expecting a change on the storefront module, but don't see file being changed.

Reply:
Well if you look at all the commits compared to develop the file is unchanged but if you just look at my most recent commit you will see I removed it.